test/puppetlabs/puppetdb/http/metrics_test.clj

```diff
@@ -37,22 +33,23 @@
   [req]
   (assoc-in req [:headers "accept"] "text/plain"))
 
-(deftestseq metrics-set-handler
+(let [app (server/build-app)]
+  (deftestseq metrics-set-handler
```

Comment:
This is similar to what we do elsewhere, such as the command_test.clj, but also I just wanted to execute the let once, not each time the deftestseq executes over the list.